test/integration/core/hash-consensus.integration.ts

```diff
@@ -0,0 +1,252 @@
+import { expect } from "chai";
+import { ZeroAddress } from "ethers";
+import { ethers } from "hardhat";
+
+import { HardhatEthersSigner } from "@nomicfoundation/hardhat-ethers/signers";
+
+import { ether, impersonate } from "lib";
+import {
+  calcReportDataHash,
+  getProtocolContext,
+  getReportDataItems,
+  ProtocolContext,
+  report,
+  waitNextAvailableReportTime,
+} from "lib/protocol";
+
+import { Snapshot, ZERO_HASH } from "test/suite";
+
+import { HashConsensus } from "../../../typechain-types";
+
+describe("Hash consensus negative scenarios", () => {
```

Comment:
```suggestion
describe("Scenario: Hash consensus negative scenarios", () => {
```

Reply:
Right now, tests are not isolated, but lit looks like they may be isolated, and then we may use beforeEach / afterEach snapshots

Reply:
They may be isolated indeed, fixed!